test/unit/core/store/index.ts

```diff
@@ -24,6 +25,9 @@ suite('Store', ({ expect, spy, stub }) => {
       const createSagaMiddleware = stub(reduxSaga, 'default').returns(sagaMiddleware);
       const createMiddleware = stub(Middleware, 'create').returns(middleware);
       const createSagas = stub(sagas, 'default').returns(['m', 'n', 'o']);
+      const isRealTimeBiasEnabled = stub(Adapter, 'isRealTimeBiasEnabled').returns(false);
+      // const persistStore = stub(persist, 'persistStore');
+      // TODO: make this work
```

Comment:
Leaving this in since this is something we should actually test.

Reply:
still need to "make this work"?
oh this is for the testing
in that case just remove it or wrap the call to `persistStore` in a function so it can be evaluated after module load